src/components/settings/TranslatorMode.svelte

```diff
@@ -38,8 +39,14 @@
     {#if $doAutoPrefix}
       <TextField
         bind:value={$autoPrefixTag}
-        label="Tag to prepend ($filterLang is replaced by your filter language)"
+        label="Tag to prepend ($filterLang is replaced by the language selected below)"
       />
+      <Dropdown
+        store={prefixTagReplacementLanguage}
+        items={languagesInfo.map((language) => language.name)}
```

Comment:
Instead of `=> language.name`, I think `=> language.lang` is intended. I think that *should* fix the auto-prefix error.